functions-methods.js

```diff
@@ -33,4 +56,18 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity (email) {
+    if (!email.includes('@') || email.includes(',') || email.lastIndexOf('.') === email.length -1) {
+        return false
+    }
+   else
+       return true
```

Comment:
Dit zou je nog wat kunnen inkorten
```suggestion
    return email.includes('@') && !email.includes(',') && email.lastIndexOf('.') !== email.length -1;
```